### ISSUE TITLE
[Question] pueue command shortcuts

### ISSUE THREAD

**Issue by sebdoesntknow** [Nukesor/pueue#245]
First of all, thanks for pueue, it's an awesome tool and I been using it for a while now. I keep my tool set pretty lean and pueue is now part of it, appreciate it.

Secondly, I'm not asking the contributors to take care of this, I'm asking them (the author) if this would be something you'd like to see as part of pueue, I would be working on it myself.

**Is your feature request related to a problem? Please describe.**

Not a problem, just thought it'd be nice to have the ability to have shortcuts for commands and sub-commands, something like this:

from `pueue add job`
to: `pueue a job`

from `pueue status`
to: `pueue s`

from `pueue remove`
to: `pueue rm`

Since I use pueue pretty often I have created an alias for it, `p` so it would be even less characters to be able to use it, i.e `p s` to get the queue status.

**Describe the solution you'd like**

I'd like to add a layer to handle the user input based on shortcuts and translate that to the full command, so the user would be able to use any of the pueue commands by typing less characters in the cli.

**Describe alternatives you've considered**

* Adding some sort of way to let the user setup their own aliases directly through pueue (like a `alias` command), i.e: `pueue alias st=status`
* Just setting up aliases on your preferred shell and leaving pueue alone :)

**Additional context**
![image](https://user-images.githubusercontent.com/4009989/133861468-1e3dd900-3044-469d-b1d8-af5edd56873d.png)

**Comment by Nukesor** [comment]
First of all, I'm happy to hear that Pueue has proven useful for you :)

It's kind of funny to see this issue, as I worked around this issue with a few aliases myself :D

```
alias p='pueue'
alias pad='pueue add --'
alias padh='pueue add -g hdd --'
alias padc='pueue add -g cpu --'
alias pst='pueue status'
alias pf='pueue follow'
```

Clap allows to easily add aliases: https://docs.rs/clap/3.0.0-beta.4/clap/struct.App.html#method.alias
E.g. in our case it would be a simple
```
    #[clap(alias("rm"))]
```

the `rm` doesn't show up in the help text yet, but, to be honest, I kind of like that.
I was kind of hesitant to to add aliases, as I feared that they would clutter the help texts and confuse people.

Though, I'm still hesitant to add aliases for everything.
Adding aliases for everything would be a bad, as having to remember too many one- or two-character aliases is quite confusing. -> `[a, cl, co, en, fo, ki, par, pau, rm, rt, rs, sta, sth, st, sw, wa]`

We should add them for those commands that are inconvenient to type and most used.

`remove` -> `rm` is definitely one that should be added.
`add` -> `a` I'm kind of torn at this one. `add` is quite fast to type. The only reason I added the alias `pad` was because of the `--` at the very end. Adding single-character aliases feels kind of dirty.
`restart` -> `re`? I definitely would appreciate that alias.

I'm against adding aliases for `start`, `pause` and `kill`. In my experience these shouldn't be done hastily. An extra quarter of a second typing gives people a little more time to overthing things :D.

**Comment by sebdoesntknow** [comment]
@Nukesor  Thanks for getting back to me! That makes a lot of sense. I'll take a peek at the code and ping you again if I have questions. Have a good one!

**Comment by Nukesor** [comment]
Resolved with b98bcb8227e0b4b8c3172255db33033de57d0e0
